_source/_posts/2020-02-03-managing-multiple-okta-instances-with-terraform-cloud.md

```diff
@@ -97,65 +110,66 @@ Before you change anything, you'll want to version control your new configuratio
 
 {% img blog/okta-terraform-cloud/terraform-cloud-create-workspace.png alt:"Create workspace using Terraform Cloud" width:"800" %}{: .center-image }
 
-On the first page of the Workspace creation flow, you need to select your source control provider. In my case, I'm using GitHub to store my code, so I'll select GitHub and grant permission for Terraform Cloud to access my project.
+On the first page of the Workspace creation flow, select **Version control flow**, as we will be saving our configuration in a Git repository. Next, you need to select your source control provider. In my case, I'm using GitHub to store my code, so I'll select GitHub and grant permission for Terraform Cloud to access my project. When GitHub prompts to select repositories to install Terraform cloud, choose one or more repositories where you intent to store Terraform configurations.
 
-{% img blog/okta-terraform-cloud/terraform-cloud-select-repository.png alt:"Terraform Cloud select repository" width:"800" %}{: .center-image }
+{% img blog/okta-terraform-cloud/terraform-cloud-select-repository.png alt:"Terraform Cloud select provider" width:"800" %}{: .center-image }
 
-Next, I'm going to select the code repository that holds my project.
+After authorizing with GitHub, click the GitHub button again to choose the repository that holds the project we created. If you haven't already, push the `okta-user-schema` project to this repository.
 
-{% img blog/okta-terraform-cloud/terraform-cloud-name-workspace.png alt:"Terraform Cloud name workspace" width:"800" %}{: .center-image }
+{% img blog/okta-terraform-cloud/terraform-cloud-select-repository-2.png alt:"Terraform Cloud select repository" width:"800" %}{: .center-image }
 
-Give your new workspace a descriptive name. In this example, we're defining our production environment, so use the repository's name with `-production` added to the end for clarity. For example, `my-project-production`.
+Give your new workspace a descriptive name. We're defining our production environment in this example, so use the repository's name with `-production` added to the end for clarity. For example, `my-project-production`.
+
+{% img blog/okta-terraform-cloud/terraform-cloud-name-workspace.png alt:"Terraform Cloud name workspace" width:"800" %}{: .center-image }
 
-Locally you defined variables in the `okta.auto.tfvars` file. Since that file isn't stored in source control, you have to explicitly define them. Click the **Variables** tab in the top navbar, then define the three variables you put into the `okta.auto.tfvars` file earlier.
+Locally you defined variables in the `okta.auto.tfvars` file. But we didn't add the `api_token` variable to that file for obvious security reasons. Hence that variable is not stored in source control; Therefore, you have to define the variable explicitly in the Terraform workspace. Click the **Variables** tab in the top navbar of the workspace, then define the `api_token` variable.
```

Comment:
Some grammar issues here with the word grouping before the semicolon and capitalization.

Should it be something like one of the following?
> But we didn't add the `api_token` variable to that file for obvious security reasons; hence the variable is not stored in source control. Therefore, you have to define...

or maybe
> But we didn't add the `api_token` variable to that file for obvious security reasons. Since the variable is not in source contr, you have to define...

Reply:
I'm not a native English speaker so I normally just go with whatever Grammarly suggests, so in this case, I'll go with your suggestion :smiley_cat:

Reply:
Grammarly is slacking! Thanks for making the change. 🤸